.github/workflows/test-upgrade.yaml

```diff
@@ -0,0 +1,135 @@
+---
+name: Kuadrant Upgrade Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      kuadrantStartVersion:
+        description: Kuadrant start version
+        required: true
+        type: string
+      clusterServer:
```

Comment:
Is having an external k8s cluster desired by these kind of tests? If not, we can just rely on kind